domain/flag/state/state.go

```diff
@@ -30,66 +30,73 @@ func NewState(factory coredb.TxnRunnerFactory, logger logger.Logger) *State {
 
 // SetFlag sets the value of a flag.
 // Description is used to describe the flag and it's potential state.
```

Comment:
**typo:** Not in your changelog but... There is actually a mispell here :) `it's` should be `its`